ports/stm32f4/supervisor/port.c

```diff
@@ -50,6 +50,7 @@ safe_mode_t port_init(void) {
     stm32f4_peripherals_gpio_init();
 
     tick_init();
+
     board_init();
```

Comment:
I think you meant to remove this.

Reply:
Weird, must have been a merge issue. Wasn't even on my local copy.

Reply:
Oh no I got it mixed up with main on my manual fix whoops